build.sbt

```diff
@@ -28,24 +28,14 @@ commands += Command.command("ci-fast") { state =>
   else "test" :: "doc" :: state
 }
 commands += Command.command("ci-slow") { state =>
-  if (isScalaJS) state // do nothing
-  else {
-    "scalahost/test:runMain scala.meta.tests.scalahost.converters.LotsOfProjects" ::
-      "testkit/test:runMain scala.meta.testkit.ScalametaParserPropertyTest" ::
-      "scalahostSbt/test" ::
-      state
-  }
+  "scalahost/test:runMain scala.meta.tests.scalahost.converters.LotsOfProjects" ::
+    "testkit/test:runMain scala.meta.testkit.ScalametaParserPropertyTest" ::
+    "scalahostSbt/test" ::
+    state
 }
 commands += Command.command("ci-publish") { state =>
-  for {
-    v <- sys.env.get("SCALA_VERSION")
-    t <- sys.env.get("TEST")
-    sjs <- sys.env.get("SCALA_VERSION")
-    if v == "2.11.8" && t == "ci-fast" && sjs == "false"
-  } {
-    RunSbtCommand("very publish")
-  }
-  state
+  if (isPublish) "very publish" :: state
```

Comment:
I suggest we rename these environment vars and associated methods to explicitly state that they are CI-related. E.g. `CI_PUBLISH` and `isCiPublish`. Otherwise, it's not quite clear whether one should bother with setting them up when trying to publish from a local machine.